HealthcareSystem/Backend/Model/DTO/BasicAppointmentSearchDTO.cs

```diff
@@ -12,23 +12,16 @@ public class BasicAppointmentSearchDTO
         public DateTime EarliestDateTime { get; set; }
         public DateTime LatestDateTime { get; set; }
 
+        public BasicAppointmentSearchDTO() {}
+
         public BasicAppointmentSearchDTO(int patientCardId, string doctorJmbg, ICollection<int> requiredEquipmentTypes, DateTime earliestDateTime, DateTime latestDateTime)
         {
             PatientCardId = patientCardId;
             DoctorJmbg = doctorJmbg;
             RequiredEquipmentTypes = requiredEquipmentTypes;
             EarliestDateTime = earliestDateTime;
             LatestDateTime = latestDateTime;
-        }
-
-        public void IsAppointmentValid()
-        {
-            if (string.IsNullOrEmpty(DoctorJmbg))
-                throw new ValidationException("Doctor jmbg cannot be null or empty.");
-            if (DateTime.Compare(EarliestDateTime,DateTime.Now) <= 0)
-                throw new ValidationException("Earliest date must be greater than " + DateTime.Now.ToShortDateString() + ".");
-            if (DateTime.Compare(LatestDateTime, DateTime.Now) <= 0)
-                throw new ValidationException("Latest date must be greater than " + DateTime.Now.ToShortDateString() + ".");
```

Comment:
You cannot delete this method, since the PatientWebApp uses it. Please fix the build issues.